new/components/vcloud/update-vcloud-vm.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -e
+
+uname -a
+
+# update system
+# yum update --skip-broken -y
+
+# install requirements for python-lxml
+yum install python-devel gcc-c++ gcc libxslt-devel libxml2-devel python-argparse zlib-devel -y
```

Comment:
Do we can add python-lxml to default cloudify virtual environment that used for create deployments? If it can be true we does not need install compilers at all.